docs/about/index.rst

```diff
@@ -17,13 +40,156 @@ Code of conduct
+`InterpretML <https://github.com/interpretml>`_.
+
+Ultimately, we hope that Fairlearn will become more than a software
+tool&mdash;a vibrant community and resource center that provides not only
+code, but also resources like domain-specific guides for when and when not to
+use different fairness metrics and unfairness mitigation algorithms.
+
+Short-term roadmap
+^^^^^^^^^^^^^^^^^^
+
+*Last update: June 19, 2020*
+
+Here we list some specific areas we would like to work on in the next few
+weeks and months. If you want to get involved, please
+:ref:`reach out <communication>`.  The inclusion of tasks in the list below
+does not mean that we are committing to deliver them by a specific date.
+It merely represents a current list of tasks that members of the community
+work on. If you are interested to work on a topic that is not covered, please
+reach out.
+
+Tasks
+~~~~~
+
+#. *Improve the `fairlearn.metrics` API*
+   More information in the
+   `fairlearn-proposals <https://github.com/fairlearn/fairlearn-proposals/blob/master/api/METRICS.md>`_
+   repository.
+
+   **How you can help:** critique the proposal, suggest new metrics, work on
+   implementing new metrics.
+
+#. *Improve the project documentation*
+   The Fairlearn website is still very new and raw with lots of gaps.
+   `This proposal <https://github.com/fairlearn/fairlearn-proposals/pull/8>`_
+   outlines the planned efforts, some of which have been completed. There are
+   still massive gaps, particularly in the user guides.
+
+   **How you can help:** critique the current content, improve writing and
+   examples, port the current documentation to the numpydoc format.
+
+#. *Improve example notebooks and add new ones*
+   The repository started out with all notebooks in the `notebooks` directory
+   as `.ipynb` files. These notebooks are now being
+   converted into `.py` files that
+   `render properly on the website <https://fairlearn.github.io/auto_examples/notebooks/index.html>`_,
+   while still being downloadable as `.py` or `.ipynb` files.
+   
+   - Some of the notebooks may be better off as parts of the user guide.
+   - As pointed out in various issues the existing notebooks have plenty of
+     potential for improvement to better reflect the sociotechnical focus of Fairlearn.
+     To accelerate this process, we experiment with weekly sessions
+     where people can discuss individual example notebooks in detail.
+   - There are only very few existing example notebooks. To cover a
+     wider range of applications we need to work with stakeholders to develop
+     more guides.
+
+   **How you can help:** critique and improve existing notebooks, join the
+   discussion sessions, create new notebooks, move the notebooks to the user
+   guide.
+
+#. *Nurture a diverse community of contributors*
+   We would like to ensure that all feel welcome and can contribute in a way
+   of their choosing. So far we have monthly developer calls.  We are
```

Comment:
Might be better as a bulleted list with links as appropriate

Reply:
That's tricky because some of these already exist, other don't or only sometimes. I had it as a list originally but then it's a sentence per bullet point which isn't really easier to read either. Added a link, though.